resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -6,7 +6,6 @@
 
 options = OpenStruct.new
 # settin' some defaults
-options.certs_package = 'ca-certificates'
```

Comment:
Do we not want that?

Reply:
It looks like in ye olde packages that was just a build dependency that got mis-translated as a runtime dependency https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb#L114-L117